capa/render/result_document.py

```diff
@@ -501,9 +510,23 @@ class MaecMetadata(FrozenModel):
     model_config = ConfigDict(frozen=True, populate_by_name=True)
 
 
+@lru_cache(maxsize=None)
+def load_rules_prevalence() -> Dict[str, str]:
+    CD = Path(__file__).resolve().parent.parent.parent
+    file = CD / "assets/rules_prevalence_data/rules_prevalence.json.gz"
+    if not file.exists():
+        raise FileNotFoundError(f"File '{file}' not found.")
+    try:
+        with gzip.open(file, "rb") as gzfile:
+            return json.loads(gzfile.read().decode("utf-8"))
+    except Exception as e:
+        raise RuntimeError(f"An error occurred while loading '{file}': {e}")
+
+
 class RuleMetadata(FrozenModel):
     name: str
     namespace: Optional[str] = None
+    prevalence: str
```

Comment:
once finalized we'll have to update the proto definition and ResultDocument (rd) files used for testing